package.json

```diff
@@ -71,6 +71,7 @@
     "eslint-plugin-svelte3": "^4.0.0",
     "jest": "^27.5.1",
     "markdown-toc": "^1.2.0",
+    "mdsvex": "^0.10.6",
```

Comment:
This is like MDX and allows us full Markdown files as route pages, or intermix Svelte with Markdown.